geo-replication/gsyncd.conf.in

```diff
@@ -220,17 +220,17 @@ template=true
 configurable=false
 
 [slave-log-file]
```

Comment:
If changed here then change also required in other places

* https://github.com/gluster/glusterfs/blob/devel/geo-replication/syncdaemon/gsyncdconfig.py#L234 and other places in this file.
* https://github.com/gluster/glusterfs/blob/devel/geo-replication/syncdaemon/resource.py

Reply:
@aravindavk  yes, this patch only changes glusterd and cli completely, plus whatever is needed to make all the geo-rep .ts ( tests/bugs/geo-replication/* and /tests/00-geo-rep/* ) pass because of these glusterd/cli changes. I tested that the .ts pass on my local machine. I am sending further changes as separate patches on top of this patch.

Reply:
ack